STM32F405RGTx/Core/Src/rtos_threads/measurements_thread.cpp

```diff
@@ -28,7 +28,7 @@ void HAL_ADC_ConvCpltCallback(ADC_HandleTypeDef* hadc) {
 void MeasurementsThread::initialize(){
     thread = RTOSThread(
         "measurements_thread",
-        1024,
+        1024*5,
```

Comment:
How did you decide on the stack size here?

Reply:
It was taken off of the bms measurements thread since we should have a similar stack

Reply:
Okie